### PR TITLE
Fix undesired messageUpdate logs when posting URLs

### PR DESCRIPTION
**Pull request by mlhynfield**

### CODE REVIEW

File: src/bot/events/messageUpdate.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = {
     }
     if (!oldMessage) return
     if (newMessage.author.bot && !global.bot.global.guildSettingsCache[newMessage.channel.guild.id].isLogBots()) return
-    if ((newMessage.content === oldMessage.content) && (newMessage.attachments.length === oldMessage.attachment_b64.split("|").filter(Boolean).length)) return // content/attachments didn't change so don't process
+    if ((escape(newMessage.content) === oldMessage.content) && (newMessage.attachments.length === oldMessage.attachment_b64.split("|").filter(Boolean).length)) return // content/attachments didn't change so don't process
```
Comment (TizzySaurus) [inline src/bot/events/messageUpdate.js:24]:
```suggestion
    # NB: Messages are stored with markdown escaped, so when comparing content we need to escape the new content
    if ((escape(newMessage.content) === oldMessage.content) && (newMessage.attachments.length === oldMessage.attachment_b64.split("|").filter(Boolean).length)) return // content/attachments didn't change so don't process

File: src/bot/events/messageUpdate.js [hunk 2]
```diff
@@ -20,7 +20,8 @@ module.exports = {
     }
     if (!oldMessage) return
     if (newMessage.author.bot && !global.bot.global.guildSettingsCache[newMessage.channel.guild.id].isLogBots()) return
-    if ((newMessage.content === oldMessage.content) && (newMessage.attachments.length === oldMessage.attachment_b64.split("|").filter(Boolean).length)) return // content/attachments didn't change so don't process
+    # NB: Messages are stored with markdown escaped, so when comparing content we need to escape the new content
```
Comment (TizzySaurus) [inline src/bot/events/messageUpdate.js]:
```suggestion
    // NB: Messages are stored with markdown escaped, so when comparing content we need to escape the new content
```

**Review by TizzySaurus — APPROVED** [review]
Thanks!

File: src/bot/events/messageUpdate.js [hunk 3]
```diff
@@ -20,7 +20,8 @@ module.exports = {
     }
     if (!oldMessage) return
     if (newMessage.author.bot && !global.bot.global.guildSettingsCache[newMessage.channel.guild.id].isLogBots()) return
-    if ((newMessage.content === oldMessage.content) && (newMessage.attachments.length === oldMessage.attachment_b64.split("|").filter(Boolean).length)) return // content/attachments didn't change so don't process
+    // NB: Messages are stored with markdown escaped, so when comparing content we need to escape the new content
+    if ((escape(newMessage.content) === oldMessage.content) && (newMessage.attachments.length === oldMessage.attachment_b64.split("|").filter(Boolean).length)) return // content/attachments didn't change so don't process
```
Comment (Elara-Discord-Bots) [inline src/bot/events/messageUpdate.js:24]: eh.. shouldn't it be 
```js
escape(newMessage.content.replace(/~/g, '\\~'), ['angle brackets'])
```
or like the one from later in the code? 
https://github.com/TizzySaurus/logger/blob/development/src/bot/events/messageUpdate.js#L64

Also it looks like for every message update it doesn't store the escaped markdown version. (even though on `MESSAGE_CREATE` it does)

**Pull request merged**